doc/htmldoc/conf.py

```diff
@@ -29,8 +29,8 @@
 from shutil import copyfile
 import json
 
-from subprocess import check_output, CalledProcessError
-from mock import Mock as MagicMock
+import subprocess
+from subprocess import check_output
```

Comment:
Why should both be necessary?! I'd expect explicit imports for all, or global import and all uses prefixed.